src/App.jsx

```diff
@@ -1,11 +1,12 @@
-import React from 'react'
+import React from "react";
+import LandingPage from "./components/LandingPage";
 
 const App = () => {
   return (
-    <div>
-      
-    </div>
-  )
-}
+    <>
+      <LandingPage />
```

Comment:
Why do you feel you have to have a separate `<LandingPage />` component? I don't see a reason but I'm ready to listen.

Reply:
We have been discussing about it with the team. The main reason its because we want to keep a clean App.jsx component as much as possible.